modules/persistent-storage-csi-vsphere-disable-storage-consequences.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-disable-storage-consequences_{context}"]
+= Consequences of disabling and enabling storage on vSphere
+
+The consequences of disabling and enabling storage on vSphere are described in the following table.
+
+.Consequences of disabling/enabling storage on vSphere
+|===
+|Disabling | Enabling
+
+a| * vSphere CSI Driver Operator un-installs the CSI driver.
+
+* Storage container orchestration (CO) should be healthy.
+
+* vSphere-problem-detector continues running, but does not emit alerts or events, and checks less frequently (once per 24 hours).
+
+* All existing persistent volumes (PVs), persistent volume claims (PVCs), storage classes, and vSphere storage policies are unchanged:
+
+** vSphere PVs cannot be used in new pods.
+
+** vSphere PVs stay mounted and attached forever to existing nodes for existing pods. These pods remain in terminating state indefinitely after deletion.
+
+* Storage classes can be removed
```

Comment:
Once https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/254 merges, then SC will be removed if you disable storage. User doesn't have to remove it.